content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,206 @@
 Italy
 =====
 
-Allow the user to generate the EDI document for Italian invoicing.
+Configuration
+=============
+The first necessary module to start booking accounting transactions on an Italian localization is
+:guilabel:`Italy - Accounting` module. Once installed it allows to:
 
-This module allows the creation of the EDI documents and the communication with the SDICoop web
-service of FatturaPA to send or receive invoices and notifications.
+- Generate electronic invoices in **FatturaPA's** format.
+- Send and receive electronic invoicing through SDIcoop.
 
 .. warning::
    Once this module is installed, it's no longer possible to send invoices via :ref:`PEC mails
```

Comment:
To be determined since the italian page is to be removed, and is the only place where PEC is documented